examples/step-50/doc/intro.dox

```diff
@@ -134,3 +191,64 @@ due to the asynchronous work "covering up" the imbalance (which assumes synchron
 However, for most realistic adaptive meshes the expectation is that this asynchronous work will
 only cover up a very small portion of the imbalance and the efficiency model will describe the
 slowdown very well.
+
+
+<h3>Workload imbalance for algebraic multigrid methods</h3>
+
+The considerations above show that one has to expect certain limits on
+the scalability of geometric multigrid methods because even in cases
+where the finest levels of a mesh are perfectly load balanced, the
+coarser levels may not be. At the same time, the coarser levels are
+weighted less (the contributions of $W_\ell$ to $W$ are small) because
+coarser levels have fewer cells and, consequently, do not contribute
+to the overall run time as much as finer levels. In other words,
+imbalances in the coarser levels may not lead to large effects in the
+big picture.
+
+Algebraic multigrid methods are of course based on an entirely
+different approach to creating a hierarchy of levels. In particular,
+they create these purely based on analyzing the system matrix, and
+very sophisticated algorithms for ensuring that the problem is well
+load-balanced on every level are implemented in both the hypre and
+ML/MueLu packages that underly the TrilinosWrappers::PreconditionAMG
+and PETScWrappers::PreconditionBoomerAMG classes. In some sense, these
+algorithms are simpler than for geometric multigrid methods because
+they only deal with the matrix itself, rather than all of the
+connotations of meshes, neighbors, parents, and other geometric
+entities. At the same time, much work has also been put into making
+algebraic multigrid methods scale to very large problems, including
+questions such as reducing the number of processors that work on a
+given level of the hierarchy to a subset of all processors, if
+otherwise processors would spend less time on computations than on
+communication.
```

Comment:
This kind of explains the point of the previous comment - all of this can also be done for the geometric multigrid method, it is just a matter of finding a volunteer. :wink:

Reply:
Yep. I've added such a comment to this paragraph.